### DISCUSSION TITLE
OS installation in an offline environment

### DISCUSSION THREAD

**Discussion by leviheichou** [coin-or/COIN-OR-OptimizationSuite#20] [Q&A] [answered]
Good morning,

Recently, I started working on a project in an offline server where I needed some of the solvers that OptimizationSuite offer, but I've been struggling constantly with the installation. I've tried using coinbrew, fetching the source code beforehand to avoid using the online features of git, but even the build process uses git somehow, so I'm completely stuck. Any ideas on how to build the code in an offline environment? Is it just impossible to use coinbrew in an offline workspace? Any additional insight would help me a lot. 

Thank you for your time and have a good day.

**Comment by tkralphs** [comment, marked as answer]
Oops, I actually already implemented an offline mode, but apparently neglected to document it. Try adding the flag `--off-line`. I just added this to the help.

**Reply by tkralphs** [reply]
If this worked, can you mark it as the answer? Thanks!
